Engine/Module/ModuleUI.cpp

```diff
@@ -113,17 +116,25 @@ void ModuleUI::RemoveComponentUI(ComponentUI* ui_to_remove)
 		delete *it;
 		ui_elements.erase(it);
 	}
+
+	SortComponentsUI();
 }
 
 void ModuleUI::InitGlyph()
 {
+	BROFILER_CATEGORY("UI: Init Glyph", Profiler::Color::HoneyDew);
 	// All functions return a value different than 0 whenever an error occurred
 	if (FT_Init_FreeType(&ft))
 		std::cout << "ERROR::FREETYPE: Could not init FreeType Library" << std::endl;
 
+#if !GAME
 	// Load font as face
 	if (FT_New_Face(ft, "Assets/Fonts/Montserrat-Light.ttf", 0, &face))
 		std::cout << "ERROR::FREETYPE: Failed to load font" << std::endl;
+#else
+	if (FT_New_Face(ft, "Library/Montserrat-Light.ttf", 0, &face))
+		std::cout << "ERROR::FREETYPE: Failed to load font" << std::endl;
```

Comment:
why we have now here std::cout?

Reply:
This is an UI issue that was merged in other branch, I fixed it in the new ui branch